src/pages/Video/video1.jsx

```diff
@@ -0,0 +1,107 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router';
+import YouTube from 'react-youtube';
+import { Navbar } from '../../components/Navbar/Navbar';
+import './video.css';
+import { videoList, notesList, playlist } from '../../data';
+import { v4 } from 'uuid';
+import { PlaylistModal } from '../../components/PlaylistModal/PlaylistModal'
+
+export function Video() {
+    let { id } = useParams();
+    const [videoDetails, setVideoDetails] = useState([]);
+
+    const [videoNotes, setNotes] = useState([]);
+
+    const [notesInput, setNotesInput] = useState("");
+
+    const [showModal, setShowModal] = useState(false);
+
+    const [videoCategory, setVideoCategory] = useState("");
+
+
+    function getVideoDetails(id) {
+        const details = videoList.find((item) => item.id === id)
+        setVideoDetails(details)
+    }
+
+    function getNote(id) {
+        const { notes } = notesList.find((item) => item.id === id)
+        // if (notes === undefined) { setNotes(videoNotes.concat({})) }
+
+        setNotes(videoNotes.concat(notes))
+    }
+
+    function handleAddNote(e) {
+        e.preventDefault();
+        setNotes(videoNotes.concat({ id: v4(), notes: notesInput }))
+    }
+
+    function handleShowModal(videoDetails) {
+        console.log(videoDetails)
+        setVideoCategory(videoDetails.category)
+
+        setShowModal(showModal => !showModal)
+    }
+
+
+
+
+
+
```

Comment:
Use vsCode extensions like remove empty lines!! It will take care of this empty lines